deploy/templates/nstemplatetiers/advanced-code.yaml

```diff
@@ -15,6 +15,20 @@ objects:
     labels:
       provider: codeready-toolchain
     name: ${USERNAME}-code
+- apiVersion: v1
+  kind: RoleBinding
+  metadata:
+    labels:
+      provider: codeready-toolchain
+    name: user-edit
+    namespace: ${USERNAME}-code
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
```

Comment:
I guess that we need to define the group and kind to differentiate between `rbac.authorization.k8s.io` and `authorization.openshift.io`, correct? Or is there anything else behind it?

Reply:
Well.. both options with 1) apiGroup and kind and 2) without them work fine. But it seems that some admission webhook modifies roleRef without apiGroup/kind because it always there after creation. There must be some reason for such admission webhook. So I decided to add it there explicitly.
Referring to the cluster role IMO makes sense regardless.